.github/workflows/performance-badges.yml

```
@@ -0,0 +1,212 @@
+          EOF
+
+          node generate-badges.js
+
+      - name: Commit and push badges
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          
+          git add badges/
+          git commit -m "Update performance badges - $(date)" || echo "No changes"
+          git push
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)